src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -296,8 +296,13 @@ public AzureAppConfigurationOptions ConfigureKeyVault(Action<AzureAppConfigurati
             var keyVaultOptions = new AzureAppConfigurationKeyVaultOptions();
             configure?.Invoke(keyVaultOptions);
 
+            if (keyVaultOptions.Credential != null && keyVaultOptions.SecretResolver != null)
+            {
+                throw new InvalidOperationException($"Cannot configure both default credentials and secret resolver for Key Vault references. Please call only one of {nameof(keyVaultOptions.SetCredential)}() or {nameof(keyVaultOptions.SetSecretResolver)}() methods.");
```

Comment:
>Please call only one of {nameof(keyVaultOptions.SetCredential)}() or {nameof(keyVaultOptions.SetSecretResolver)}() methods. [](start = 142, length = 123)

We could remove the opening and closing parentheses, similar to the error message in the following link.
[https://github.com/Azure/AppConfiguration-DotnetProvider/blob/1d2a0cb24f0e193907211be4e6d8df0d0a47606e/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationSource.cs#L50](https://github.com/Azure/AppConfiguration-DotnetProvider/blob/1d2a0cb24f0e193907211be4e6d8df0d0a47606e/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationSource.cs#L50)

Suggestion : "Please call either {nameof(keyVaultOptions.SetCredential)} or {nameof(keyVaultOptions.SetSecretResolver)} method."